src/util.py

```diff
@@ -296,3 +303,79 @@ def make_byr_cmap():
     warnings.filters.pop(0)
 
     return cmaps.BlueYellowRed
+
+
+def set_vector_density(ds, lat_density=1, lon_density=1, minDistance=0):
+    """
+    Utility function to change density of vector plots.
+
+    Ars:
+
+        da (:class:`xarray.core.dataarray.DataArray`):
+            Data array that contains the vector plot data.
+
+        lat_density (:class:`int`):
+            Value in range (0,1] that determines the density of the vectors in the y range.
+
+        lon_density (:class:`int`):
+            Value in range (0,1] that determines the density of the vectors in the x range.
+
+        minDistance (:class:`int`):
+            Value in degrees that determines the distance between the vectors.
+
+    """
+    import math
+
+    if minDistance != 0 and lat_density != 1 or lon_density != 1:
+        raise Exception("minDistance and lat/lon_density parameters cannot be used simulatenously.")
+
+    # Change the density with parameter "minDistance"
+    if minDistance != 0:
+
+        lat_every = 1
+        lon_every = 1
+
+        # Get distance between points in latitude (y axis)
+        lat = ds.U['lat']
```

Comment:
Could you add a kwarg to allow the user to specify what variables should be affected by this function? Not every NetCDF file will have the same naming conventions. For example, on line 156 of `NCL_overlay_6.py` the variables `u`, `v`, and `t` are thinned out in the same way in order to reduce the density of the vectors, but this function wouldn't currently work for that application since the variable names are hard coded here.